docs/api/url.md

```diff
@@ -0,0 +1,86 @@
+# 🔗 URL
+An API for working with HTTP URLs. 
+
+## Quick Start
+
+We can construct a url in two ways, either as an absolute url or a relative url.
+```typescript 
+let url : URL = new URL(`tezos://${my_contract.address}/entrypoint`);
+let url2 : URL = new URL('../entrypoint_2', url.href);
+```
+The constructor will raise and exception if the arguments cannot be parsed into a valid url.
+The [`URL.canParse()`](#canParse) method can be used to check if the arguments can be parsed correctly.
+```typescript
+if (URL.canParse(foo, bar)) {
+  let url = new URL(foo, bar)
+  console.log(url.href)
+} else {
+  // the URL cannot be parsed, take appropriate action.
+  console.error("Invalid URL")
+}
+```
```

Comment:
We should take the opportunity (here initially) to explain the `tezos://` scheme